js/tic_tac_toe.js

```diff
@@ -1,2 +1,69 @@
-// Your code here
-// Use the jQuery
+$(document).ready(function(){
+  $winner = $('#winning');
+  $tiles = $('td');
+  $tiles.map(function(){
+    this.player = NaN;
+    this.set = false;
+  });
+  $tiles.on('click',Circle.changeColor);
+});
+
+
+var Circle = (function(){
+  var click = 0;
+  var changeColor = function(){
+    if(click % 2 === 0 && $(this).get(0).set === false){
+      $(this).css('background-color', '#f00');
+      $(this).get(0).player = 'Red';
+      $(this).get(0).set = true;
+      click++;
+    }else if ($(this).get(0).set === false){
+      $(this).css('background-color', '#000');
+      $(this).get(0).player = 'Black';
+      $(this).get(0).set = true;
+      click++;
+    }
+
+    _checkRows();
+    _checkColumns();
+    _checkDiagonals();
+  };
+  var _checkRows = function(){
+    if(_threeIn(0,1,2)){
```

Comment:
Its a little verbose, but at the same time well structured and flexible. Well done!